_data-prepper/pipelines/configuration/sources/otel-logs-source.md

```diff
@@ -51,18 +51,133 @@ To get started, create a `pipeline.yaml` file and add `otel_logs_source` as the
 
 ```yaml
 source:
-    - otel_logs_source:
+  otel_logs_source:
 ```
+{% include copy.html %}
 
 To generate data in the OpenTelemetry format, set the `output_format` setting to `otel`, as shown in the following example:
 
 ```yaml
 source:
-    - otel_logs_source:
-        output_format: otel
+  otel_logs_source:
+    output_format: otel
+```
+{% include copy.html %}
+
+## Example
+
+The following pipeline demonstrates Data Prepper receiving OTLP logs over HTTPS using a PEM cert and key with unframed HTTP at a custom path, accepting `gzip` payloads, preserving OTel shaped documents, and indexing them into OpenSearch:
```

Comment:
I think the phrase "preserving OTel shaped documents" may be unclear. I think you are referring to the `output_format: otel`. Is there a better way to express this.

Reply:
@dlvenable I've updated this now, can you have another look please